Makefile

```
@@ -202,3 +202,6 @@ JSONNETFMT_CMD := $(JSONNETFMT) -n 2 --max-blank-lines 2 --string-style s --comm
 .PHONY: jsonnet-fmt
 jsonnet-fmt: | $(JSONNETFMT)
 	PATH=$$PATH:$(BIN_DIR):$(FIRST_GOPATH)/bin echo ${JSONNET_SRC} | xargs -n 1 -- $(JSONNETFMT_CMD) -i
+
+generate-rules-code: $(OAPI_CODEGEN)
```

Let's make this a concrete target that produces a real file, rather than a phony target (or make it phony for convenience in addition to the real file). Let's also add this target as a dependency to the main API binary.
```suggestion
rules/rules.go: $(OAPI_CODEGEN)
```

---

added (also had to make it .PHONY so the file would be correctly updated when running `make rules/rules.go` or `make generate`)

regarding add as a dependency from the main API bin, you mean here?
https://github.com/observatorium/api/blob/d1f53ca3b69122164d0db2097d224f5391b862f4/Makefile#L55

I think the wildcard would already catch the `rules/rules.go` file?